wordpress-mobile/release-toolkit
# [Toolkit Cleanup] Yard doc: step 2 – rename module Helpers -> Helper

This PR sits on top of https://github.com/wordpress-mobile/release-toolkit/pull/198 and simply renames the `module Helpers` into `module Helper` for consistency both in code and in the generated documentation.

> ⚠️  This PR is ready for review but PLEASE DO NOT MERGE it yourself. Once it's approved I will take care of merging it as it should only be merged after #198 is itself merged.

### Why?

* In fastlane's own source code, the helpers live inside `Fastlane::Helper` (singular).
* In our codebase, some helpers are named under `Fastlane::Helper` but others are named `Fastlane::Helpers`.

This discrepancy makes the code inconsistent but also makes the YARD documentation incoherent, by splitting some modules and helpers in one module and other helpers in another.

### How?

This is mainly a "Find & Replace All" on the codebase for `module Helpers` and `Fastlane::Helpers` to its singular form instead. I made it a dedicated PR for easier review because it touches many files and already makes quite a big diff all by itself.

### To Test

* Review the code changes
* Run `bundle install && bundle exec rspec` to run the existing tests and ensure they still compile and pass
* Optionally you could try running a couple of actions (`bundle exec fastlane run <action_name>` to ensure that they still work, though be careful about potential side effects triggered by those actions (we wouldn't want to cut a release or push a new beta unexpectedly)
* Run `bundle exec yard doc` to generate the documentation. Open `yard-doc/index.html` in your browser and ensure that the modules and helpers are consistently classified under a common `Fastlane` > `Helper` hierarchy
